fetch_data.py

```diff
@@ -55,21 +74,26 @@ def fetch_users(apis, target, are_users, nodes_to_consider, max_tweets_count, ou
     :return: followers, friends, intersection of both, and union of both
     """
     followers = friends = []
-    if not are_users:
-        tweets = get_or_set(out_path / target / tweets_file,
-                            partial(fetch_tweets, search_query=target, api=apis[0], max_count=max_tweets_count),
-                            api_function=True)
-        print("Found {} tweets.".format(len(tweets)))
-        followers = [{**tweet["user"], "query_created_at": tweet["created_at"]} for tweet in tweets]
-        print("Found {} unique authors.".format(len(set(fol["id"] for fol in followers))))
-        get_or_set(out_path / target / followers_file, followers, api_function=False)
-    else:
+    if mode == Mode.USERS:
         if nodes_to_consider in ["followers", "all"]:
             followers = fetch_users_paged(apis, target, api_func='GetFollowersPaged',
                                           out_file=out_path / target / followers_file)
         if nodes_to_consider in ["friends", "all"]:
             friends = fetch_users_paged(apis, target, api_func='GetFriendsPaged',
                                         out_file=out_path / target / friends_file)
+    else:
+        if mode == Mode.TWEETS:
+            tweets = get_or_set(out_path / target / tweets_file,
+                            partial(fetch_tweets, search_query=target, api=apis[0], max_count=max_tweets_count),
+                            api_function=True)
+        else: # mode == Mode.LIKES
```

Comment:
I would rather use `elif mode == Mode.LIKES`, and add an else: raise `ValueError("Unknown mode")` in the end.

Reply:
Done.